WPAndroid Integration.md

```diff
@@ -0,0 +1,113 @@
+Building the project as such will make WPAndroid build the source code for `react-native-bridge`, `react-native-aztec` and other sublibraries from the locally checked out folder.
+It'll also expect the metro server to be running to fetch the JS bundle.
+
+- [Work with local gutenberg-mobile checkout](#work-with-local-gutenberg-mobile-checkout)
+- [Deploy a new version of `react-native-bridge` from CI](#deploy-a-new-version-of-react-native-bridge-from-ci)
+- [Manually deploy a new version of `react-native-bridge`](#manually-deploy-a-new-version-of-react-native-bridge)
+- [Update WPAndroid's binary version](#update-wpandroids-binary-version)
+- [How to](#how-to)
+    - [How to test a gutenberg-mobile PR in WPAndroid](how-to-test-a-gutenberg-mobile-pr-in-wpandroid)
+    - [How to share WIP WPAndroid APK](how-to-share-wip-wpandroid-apk)
+    - [How to checkout gutenberg-mobile to deployed binary version](#how-to-checkout-gutenberg-mobile-to-deployed-binary-version)
+
+---
+
+### Work with local gutenberg-mobile checkout
+
+- Make sure you run `npm install` or `npm ci`
+- Copy [local-builds.gradle-example](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/local-builds.gradle-example) renaming it to `local-builds.gradle`
+- Update `localGutenbergMobilePath`to your local `gutenberg-mobile`checkout
+- Run metro server with `npm run start:reset`
+
+#### How it works:
+
+When [settings.gradle](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/settings.gradle) finds `localGutenbergMobilePath` in `local-builds.gradle`, it'll substitute the below binary dependency with the local folder:
+
+```
+implementation "$rootProject.gradle.ext.gutenbergMobileBinaryPath:$rootProject.ext.gutenbergMobileVersion"
+```
+
+`react-native-bridge` is setup to include all the other projects, so no other action is necessary.
+
+---
+
+### Deploy a new version of `react-native-bridge` from CI
+
+There are a few different ways to do this:
+
+1. Open a new PR or push a new commit to an open PR which will be published as `<PR number>-<commit full SHA1>`
+2. Merge a PR to `develop` which will be published as `<develop>-<commit full SHA1>`
+3. Create a new tag which will be published as `<tag name>`
+
+**How it works:**
+
+CI will run the following commands:
+
+- `npm ci`
+- `npm run bundle:android`
+- `gutenberg/packages/react-native-bridge/android/publish-aztec-and-bridge.sh <version>`
+
+---
+
+### Manually deploy a new version of `react-native-bridge`
+
+_**IMPORTANT:** This should only be used for testing.
+We shouldn't commit a manually deployed version to WPAndroid's `develop` because the process bypasses the guarantees CI would otherwise give us._
+
+- Make sure you run `npm install` or `npm ci` first
+- Run `npm run bundle:android` so the JS bundle is created
+- Run `publish-aztec-and-bridge.sh` script in `react-native-bridge`, giving it a version name such as `locally-built-by-<your_name>-<commit full SHA1>`
```

Comment:
> giving it a version name

Might be good to be explicit about what "it" is and how you give it a version name, i.e. is this something the script prompts you for or an argument to the script?

Reply:
I've reworked this in 6a5ca7effc502ea798275d8ced92a54755d14942, let me know if that's clearer.

FYI: The script will also return an example usage if it's not called properly.

Reply:
Looks great! 👍